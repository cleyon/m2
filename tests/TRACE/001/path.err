m2trace: Path search for 'incl2' found 'path_search/incl2'
