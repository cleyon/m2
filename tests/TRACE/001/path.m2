@define __TRACE__ 1
@tracemode +p
@include incl2
Again
@tracemode -p
@include incl2
Done
