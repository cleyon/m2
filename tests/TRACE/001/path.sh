M2="$1"
file="$2"
${M2} I=path_search ${file}
