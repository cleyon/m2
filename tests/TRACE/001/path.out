path_search/Include 2
Again
path_search/Include 2
Done
